app/controllers/staff/reports_submissions_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Staff::ReportsSubmissionsController < Staff::BaseController
+  include Secured
+
+  def show
+    report = Report.find(params[:report_id])
+    authorize report
+    @report_presenter = ReportPresenter.new(report)
+    render "staff/reports/submit"
+  end
+
+  def update
+    @report = Report.find(params[:report_id])
+    authorize @report
+
+    @report.state = :submitted
+    if @report.valid?
+      @report.save!
+      @report.create_activity key: "report.submitted", owner: current_user
+      redirect_to complete_report_submit_path(@report)
```

Comment:
How about rendering the "completed" page here rather than redirecting?

Reply:
Even better! Thank you! :)